themes/Arch-Colors/lxqt-panel.qss

```diff
@@ -179,7 +179,128 @@ QMenu::indicator:non-exclusive:checked {
 
 #MainMenu ActionView::item:selected {
     background: #1899da;
-    color: white
+    color: white;
+}
+
+/*
+ * Fancymenu
+ */
+#FancyMenu {
+    qproperty-icon: url(mainmenu.svg);
+    background: #162635;
+    margin: 0px;
+    padding: 3px;
+    color: #e7eff1;
+}
+
+LXQtFancyMenuWindow{
+    background: #041b27;
+}
+
+#FancyMenu:hover {
+    color: #d7e3e6;
+}
+
+#FancyMenu  #AppView {
+    background: #06293a;
+    border: 1px solid grey;
+}
+
+#FancyMenu QListView::item {
+    color: #d7e3e6;
+    padding: 5px 30px 5px 3px;
+}
+
+#FancyMenu QListView::item:hover {
+    background: #003d5b;
+    color: #d7e3e6;
+    padding: 0px;
+}
+
+#FancyMenu QListView::item:pressed {
+    text-decoration: none;
+}
+
+#FancyMenu QListView::item:selected {
+    background: #1899da;
+    color: white;
+}
+
+#FancyMenu QLineEdit {
+    color: white;
+    background:  #041b27;
+    padding: 3px;
+}
```

Comment:
By not setting the border here, it's left to the widget style. It may seem OK with Fusion but can be quite weird with Kvantum (and, perhaps, some other styles). Please add it like this (I chose it from the theme itself, but could change it if you want):

```css
#FancyMenu QLineEdit {
    color: white;
    background:  #041b27;
    padding: 3px;
    border: 1px solid #d7e3e6;
}
```

Reply:
> border: none;

You prefer no border with the search entry? It's OK with me if you do -- just asked to know if it's intentional.

Reply:
There was a border in theme colors, but it was drawn by the widget. It's intentional.